lenskit-core/src/main/java/org/grouplens/lenskit/data/event/Like.java

```diff
@@ -20,14 +20,120 @@
  */
 package org.grouplens.lenskit.data.event;
 
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.io.Serializable;
+
 /**
  * Generic interface representing a unary interaction.  This provides a basic, generic representation
  * of likes, clicks, purchases, and other unary interactions.  Applications may want to implement
  * their own events to represent such interactions.
+ * <p>
+ * This class can be created with {@link LikeBuilder} or the {@link #create(long, long, long)} method.
```

Comment:
Not that I think its a big issue, but do we _need_ this many ways to make a like? this seems a little excessive.